src/components/PageSizeSelector.tsx

```diff
@@ -38,20 +41,38 @@ export const PageSizeSelector: React.FC<Props> = ({
       sendAnalyticsEvent();
     });
 
-  return (
+  return useLeafygreen ? (
```

Comment:
I like this pattern! It should make the conversion process easier down the line 🎉